js/script.js

```diff
@@ -0,0 +1,55 @@
+const books = JSON.parse(localStorage.getItem('books')) || [];
+
+function renderBooks() {
+  const listDisplay = document.getElementById('list');
+  listDisplay.innerHTML = '';
+
+  books.forEach((book, index) => {
+    const bookDiv = document.createElement('div');
+    bookDiv.className = 'book';
+    const spanTitle = document.createElement('span');
+    spanTitle.textContent = book.title;
+    const spanAuthor = document.createElement('span');
+    spanAuthor.textContent = book.author;
+
+    const removeBtn = document.createElement('button');
+    removeBtn.className = 'btn';
+    removeBtn.textContent = 'Remove';
+    /* eslint-disable */
+    removeBtn.addEventListener('click', () => removeBook(index));
+    /* eslint-enable */
+    bookDiv.appendChild(spanTitle);
+    bookDiv.appendChild(spanAuthor);
+    bookDiv.appendChild(removeBtn);
+
+    listDisplay.appendChild(bookDiv);
+  });
+}
+
+function addBook() {
+  const titleInput = document.getElementById('title');
+  const authorInput = document.getElementById('author');
+  const title = titleInput.value;
+  const author = authorInput.value;
+
+  if (title && author) {
+    const bookData = {
+      title, author,
+    };
+    books.push(bookData);
+    localStorage.setItem('books', JSON.stringify(books));
+    titleInput.value = '';
+    authorInput.value = '';
+    renderBooks();
+  }
+}
+
+function removeBook(index) {
+  books.splice(index, 1);
+  localStorage.setItem('books', JSON.stringify(books));
+  renderBooks();
+}
```

Comment:
- [OPTIONAL] You're doing great with manipulating arrays! I'd like to propose using the `filter` method instead of the `splice` method for removing elements. The `filter` method returns a new array containing elements that meet a certain condition, which makes your code more functional and less prone to unexpected side effects. By avoiding direct mutation of the original array like `splice` does, you ensure a safer and more predictable outcome. Give `filter` a shot, and you'll likely find your code easier to understand and maintain. Keep up the good work!